lib/coverband/utils/result.rb

```diff
@@ -55,11 +55,16 @@ def created_at
       # the line-by-line coverage to zero (if relevant) or nil (comments / whitespace etc).
       def self.add_not_loaded_files(result, tracked_files)
         if tracked_files
+          # TODO: Can we get rid of this dup it wastes memory
           result = result.dup
           Dir[tracked_files].each do |file|
             absolute = File.expand_path(file)
 
-            result[absolute] ||= Coverband::Utils::LinesClassifier.new.classify(File.foreach(absolute))
+            puts "hit #{absolute} "*60 if result[absolute].nil?
+            result[absolute] ||= {
+              'data' => Array.new(File.foreach(absolute).to_a.length) { 0 },
```

Comment:
Got interested in File#foreach. Looks like we can do a count on the enumerator instead of converting to array.

```ruby
Array.new(File.foreach(absolute).count)
```

Reply:
nice good call